blog/_posts/2020-05-04-april-commit-of-the-month.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: April-2020-commit-of-the-month
+date: 2020-05-04
+author: dmiller
+layout: blog
+title: "Using Pack and Buildpacks"
+subtitle: "Commit of the Month: A Buildpack Extension"
+image: /assets/images/april-2020-commit-of-the-month/featuredImage.jpg
+image_caption: "Photo by Iris Aldeguer on <a href='https://www.flickr.com/photos/irisux/4372485391'>Flickr</a>."
+image_type: "contain"
+tags:
+  - tilt
+  - cotm
+  - extensions
+keywords:
+  - tilt
+  - pack
+  - buildpack
+---
+
+In April we introduced [Tilt Extensions](https://docs.tilt.dev/extensions.html). Extensions are open-source packaged functions that extend the capability Tilt, right inside your Tiltfile. Since releasing them we've seen several great extensions contributed by members of the community and we're highlighting one of them as April's Commit of the Month: [an extension to use pack and buildbacks to build container images](https://github.com/windmilleng/tilt-extensions/commit/e1d193e508ce8468800d0985ee4714aa65d78c87) by [Gareth Rushgrove](https://twitter.com/garethr)!
+
+## What is buildpack? What is pack?
```

Comment:
should this be:
- what is `buildpack`
- what is a buildpack
- what is a `buildpack`
- what are buildpacks?

I feel like `pack` as a CLI tool should always be in backticks, here and in the header below

Reply:
ah yeah good call